ironfish-cli/src/command.ts

```diff
@@ -167,6 +170,11 @@ export abstract class IronfishCommand extends Command {
       configOverrides.logLevel = '*:verbose'
     }
 
+    const jsonLogsFlag = getFlag(flags, JsonLogsFlagKey)
+    if (typeof jsonLogsFlag === 'boolean' && jsonLogsFlag !== JsonLogsFlag.default) {
+      configOverrides.jsonLogs = jsonLogsFlag
+    }
+
```

Comment:
Only a computer would want this right? Why would a computer not set the config first and use this field?

Reply:
In the use case we want to get all the node logs from startup as JSON. When starting up the nodes datadirs aren't initialized yet and I don't think we have an option to pass in a config file on startup. We could startup then change the config but then we'd have to restart the node or miss some logs. 

I think it could also be useful to encode for terraform scripts if we want to log to docker or something and don't want to have to make sure the config is set correctly in the node's data dir